omnibus/package-scripts/agent/prerm

```diff
@@ -82,15 +82,33 @@ remove_py_compiled_files()
 remove_downloader_data()
 {
     # Remove all the data that was downloaded using the integration command
+    # This is purely for cleaning up after ourselves
     DOWNLOADER_DATA="$INSTALL_DIR/embedded/lib/python2.7/site-packages/datadog_checks/downloader/data"
     if [ -d $DOWNLOADER_DATA ]; then
         rm -rf $DOWNLOADER_DATA
     fi
 }
 
+uninstall_integrations()
+{
+    # Uninstall every datadog integration
+    PIP_PATH=$INSTALL_DIR/embedded/bin/pip
+    if [ -x $PIP_PATH ]; then
+        echo "Uninstalling integrations..."
+        $PIP_PATH freeze | grep ^datadog- | grep -v datadog-checks-base | xargs $PIP_PATH uninstall -y -q --no-cache-dir || true
+    fi
+}
+
 if [ -f "/etc/debian_version" ] || [ "$DISTRIBUTION" = "Debian" ] || [ "$DISTRIBUTION" = "Ubuntu" ]; then
     stop_agent
     deregister_agent
+    # Starting with 6.10, integrations are uninstalled on package removal
+    # Upgrade from 6.8 or 6.9 to 6.10 won't uninstall integrations because it was previously done in preinst
+    # It might cause weird issues with pip and the integration command
+    # if several integration wheel metadata folders are present (pip wouldn't detect the right version of an integration),
+    # but they would be fairly easy to solve by manually removing them
```

Comment:
It'd be nice to reflect this comment in the preinst script. And here it could make sense to put it in the function body.

(to avoid too many inline comments, a link to a PR that describes the issue in details would work as well)